src/azure-cli/azure/cli/command_modules/profile/custom.py

```diff
@@ -78,7 +78,7 @@ def get_access_token(cmd, subscription=None, resource=None, scopes=None, resourc
         'tokenType': creds[0],
         'accessToken': creds[1],
         # 'expires_on': creds[2].get('expires_on', None),
-        'expiresOn': creds[2].get('expiresOn', None),
```

Comment:
Why do you need to change this line?

Reply:
We need to guarantee that `expiresOn` always exists in `creds[2]`.